### PR TITLE
port to oculus mobile sdk 0.6.2

### PR DESCRIPTION
**Pull request by thomasflynn**
port to oculus mobile sdk 0.6.2

GearVRf-DCO-1.0-Signed-off-by: Tom Flynn
tom.flynn@samsung.com

### CODE REVIEW

**Comment by chenchao1407** [comment]
Well. It seems to me that you will have to update all samples' xml to match the configuration change.

**Comment by thomasflynn** [comment]
good point.  fortunately, the default will happen in the meantime for the resolutionWidth/Height parameters.  i'll go modify the existing xml.

**Comment by thomasflynn** [comment]
modified the xmls.  didn't take as long as i thought.  thank you bash, sed, and regular expressions :)

**Comment by dknaan** [comment]
Looks good. Works fine on my Note4 Mali and S6.

File: GVRf/Framework/src/org/gearvrf/GVRXMLParser.java [hunk 1]
```diff
@@ -183,11 +183,16 @@
                             } else if (attributeName.equals("resolveDepth")) {
                                 settings.eyeBufferParms.enableResolveDepth(Boolean
                                         .parseBoolean(attributeValue));
-                            } else if (attributeName.equals("resolution")) {
+                            } else if (attributeName.equals("resolutionWidth")) {
                                 int resolution = Integer
                                         .parseInt(attributeValue);
```
Comment (chenchao1407) [inline GVRf/Framework/src/org/gearvrf/GVRXMLParser.java]: A little bit picky here, the name should be resolutionWidth and resolutionHeight here

File: GVRf/Framework/src/org/gearvrf/utility/VrAppSettings.java [hunk 2]
```diff
@@ -205,22 +208,41 @@ public void setColorFormat(ColorFormat colorFormat) {
...
+        /**
+         * Set resolution width.
+         * 
+         * @param resolution
+         *            Resolution width to set.
          */
-        public int getResolution() {
-            return resolution;
+        public void setResolutionWidth(int resolution) {
+            this.resolutionWidth = resolution;
```
Comment (chenchao1407) [inline GVRf/Framework/src/org/gearvrf/utility/VrAppSettings.java]: int resolutionWidth

**Comment by chenchao1407** [comment]
Overall is fine, tested and worked well. Just a little issue on naming. And also, with gvrexposeapisample, there are more than one xmls which should be modified to match "resolution to resolutionWidth/Height" change.

Thanks

**Comment by thomasflynn** [comment]
update per Chao's comments.

**Comment by thomasflynn** [comment]
ok, updated.  did i miss anything else?

**Comment by chenchao1407** [comment]
As far as I can see, this one is good to go

**Comment by thomasflynn** [comment]
any further comments on this?

**Comment by jason2kim** [comment]
looks good to me too

**Comment by dknaan** [comment]
Not on my side. It worked for me well yesterday.

**Pull request merged**